packages/web3/src/version.ts

```diff
@@ -1 +1 @@
-/* eslint-disable header/header */ export const Web3PkgInfo = { version: '4.0.0-alpha.1' };
+/* eslint-disable header/header */ export const Web3PkgInfo = { version: '4.0.0-alpha.0' };
```

Comment:
Its auto updated by reading https://github.com/ChainSafe/web3.js/blob/7a55b62bfa1c34b78a2014cba97e2e02dff353f9/packages/web3/package.json#L3 so web3 package;s package.json should also be updated